contracts/src/OffChainMessageContracts/README.md

```diff
@@ -0,0 +1,15 @@
+## Off Chain Message Contracts
+
+This directory contains Solidity contracts that use off chain [Avalanche Warp Messages](https://docs.avax.network/build/cross-chain/awm/overview). These are Warp messages whose source contract address is the zero address that is not owned and therefore could not have been originated on-chain.
```

Comment:
```suggestion
This directory contains Solidity contracts that use off-chain [Avalanche Warp Messages](https://docs.avax.network/build/cross-chain/awm/overview). These are Warp messages that are signed via manual approve by a chain's validators, rather than being submitted on-chain to be sent.
```

Nit: Technically off-chain messages don't _need_ to have source address of zero, that's just one way a contract receiving them can be sure that they couldn't have been published on-chain.